rcl/src/rcl/rmw_implementation_identifier_check.c

```diff
@@ -55,7 +55,8 @@ extern "C"
   static void f(void)
 #endif
 
-INITIALIZER(initialize) {
+rcl_ret_t _internal_rmw_check(void)
```

Comment:
@Blast545 nit: if this is going to be public, perhaps:
```suggestion
rcl_ret_t rcl_rmw_implementation_check(void)
```
is a bit better.